src/libraries/System.Reflection.Metadata/tests/Metadata/TypeNameParserHelpersTests.cs

```diff
@@ -68,22 +68,8 @@ public void GetAssemblyNameCandidateReturnsExpectedValue(string input, string ex
         public void AppendRankOrModifierStringRepresentationAppendsExpectedString(int input, string expected)
         {
             ValueStringBuilder builder = new ValueStringBuilder(initialCapacity: 10);
-            Assert.Equal(expected, TypeNameParserHelpers.GetRankOrModifierStringRepresentation(input, ref builder));
-        }
-
-        [Theory]
-        [InlineData(typeof(List<int>))]
```

Comment:
Are we losing some test coverage here?

Reply:
This unit test was testing the helper method that was used to produce the result of `TypeName.FullName`. We do have plenty of tests that test the latter:

https://github.com/dotnet/runtime/blob/109c9469a7f76bdc98300bd9c7bb4eaf014be971/src/libraries/System.Reflection.Metadata/tests/Metadata/TypeNameTests.cs#L1030

https://github.com/dotnet/runtime/blob/109c9469a7f76bdc98300bd9c7bb4eaf014be971/src/libraries/System.Reflection.Metadata/tests/Metadata/TypeNameTests.cs#L487